components/builder-db/src/config.rs

```diff
@@ -49,7 +53,11 @@ impl Default for DataStoreCfg {
                        connection_retry_ms:    300,
                        connection_timeout_sec: 3600,
                        connection_test:        false,
-                       pool_size:              (num_cpus::get() * 2) as u32, }
+                       pool_size:              (num_cpus::get() * 2) as u32,
+                       ssl_mode:               None,
```

Comment:
Could you also please add a test for these configs here - https://github.com/habitat-sh/builder/blob/master/components/builder-api/src/config.rs#L260